### DISCUSSION TITLE
Time on the Moon isn't consistent

### DISCUSSION THREAD

**Discussion by StalwartBlue** [Stellarium/stellarium#2303] [Q&A] [answered]
I'm using Stellarium for my astronomy class, and it's been pretty amazing.  My latest assignment has my students observing the lunar eclipse on 5/26/2021, first from the Earth and then from the Moon.  

I'm having the students record the time for various parts of the lunar eclipse, but a couple of student (both on Macs) are getting different times.

Details:
Students start by setting the time to 2 am.  When we go the Moon, as soon as we open the Navigation window (to change the location to 0 lat, 0 long), the time automatically changes from 2 am to 9 am.  I'm in California, so I assume the time is shifting Pacific Time to Universal Time.  But two of my students (both on Macs) have the time still at 2 am when they go to the Moon.  

I'm going to have them add 7 hours to their time so they can get the correct answers for their observations, but if there is a better work-around, I'd love to hear about it.

**Comment by gzotti** [comment, marked as answer]
Don't assume a time zone. Make sure it is the right one by checking it (see the entry "UTC&pm;xx:xx" in the status line at screen bottom) or even actively setting it. F6 (location panel), bottom right. Set to UTC for a "neutral" result or whatever timezone you need.

**Reply by StalwartBlue** [reply]
Thanks!  That was exactly the information I needed.
